proto/elys/incentive/params.proto

```diff
@@ -9,6 +9,6 @@ option go_package = "github.com/elys-network/elys/x/incentive/types";
 // Params defines the parameters for the module.
 message Params {
   option (gogoproto.goproto_stringer) = false;
-	repeated IncentiveInfo LPIncentives = 1 [(gogoproto.nullable) = false];
-	repeated IncentiveInfo StakeIncentives = 2 [(gogoproto.nullable) = false];
+	repeated IncentiveInfo lp_incentives = 1 [(gogoproto.nullable) = false];
+	repeated IncentiveInfo stake_incentives = 2 [(gogoproto.nullable) = false];
```

Comment:
@kenta-elys Did we decide not to use the x/tokenomics InflationEntry type for this?

Reply:
I have a thought to fetch only amount data from tokenomic module. I could change to use tokenomic inflationentry no problem. Could you please focus on checking the formula that i have used to calculate the uncommitted token amount. And also the eden and eden boost token handling?

Reply:
this change will be approached in a later ticket cc @kenta-elys @AustinoBombino